assets/queries/azureResourceManager/web_app_not_using_tls_last_version/query.rego

```diff
@@ -0,0 +1,43 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Web/sites"
+	not is_last_tls(value)
+
+	issue := prepare_issue(value)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("%s.name=%s%s", [common_lib.concat_path(path), value.name, issue.sk]),
+		"issueType": issue.issueType,
+		"keyExpectedValue": "'siteConfig.minTlsVersion' is 1.2",
+		"keyActualValue": issue.keyActualValue,
+	}
+}
+
+is_last_tls(resource) {
+	resource.properties.siteConfig.minTlsVersion == "1.2"
+}
+
+prepare_issue(resource) = issue {
+	_ = resource.properties.siteConfig.minTlsVersion
```

Comment:
I'm not sure if it will ever reach this point since it was denied on line 25 and 11. Can we just make an OR statement?